t/zmq.t

```diff
@@ -34,7 +34,7 @@ subtest 'receive msg' => sub {
                 }
                 zmq_sendmsg($sock, zmq_msg_init_data("last"), 0);
             }
-            exit 0;
+            zmq_term($ctxt);
```

Comment:
**nitpick** what `zmq_term` do & how its better than `exit` ?

Reply:
this is just the correct way of close it to avoid failing tests like:

https://www.cpantesters.org/cpan/report/0d6a849c-d55c-11ed-ba78-a24819850bc8
```
panic: bad free, ->next->prev=56132b67c610, header=56132b67c610, ->prev->next=0 at t/zmq.t line 37.
    # No tests run!

#   Failed test 'No tests run for subtest "receive msg"'
#   at t/zmq.t line 58.
# Looks like you failed 1 test of 5.
```